### PR TITLE
{chem,devel}[GCCcore/11.3.0] Molden v7.1, makedepend v1.0.7

### PR DESCRIPTION
**Pull request by maxim-masterov**

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/673bf77fb29356f151092acd15b2e168 for a full test report.

File: easybuild/easyconfigs/m/Molden/Molden-7.1-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+easyblock = 'MakeCp'
+
+name = 'Molden'
+version = '7.1'
+
+homepage = 'https://www3.cmbi.umcn.nl/molden'
```
Comment (branfosj) [inline easybuild/easyconfigs/m/Molden/Molden-7.1-GCCcore-11.3.0.eb:6]:
```suggestion
homepage = 'https://www.theochem.ru.nl/molden/'
```
I get a 404 for the original. I think the suggested one is the correct replacement.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/Molden/Molden-7.1-GCCcore-11.3.0.eb (hunk 1 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/m/Molden/Molden-7.1-GCCcore-11.3.0.eb]: done

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Going in, thanks @maxim-masterov!

**Comment by yanchenmochen** [comment]
I tryied to install Molden 7.1 using "makedepend-1.0.7-GCCcore-11.3.0.eb", but failed because of 
```
make -C src/ambfor FC=gfortran LDR="gfortran -g -m64" EXT=""
make[1]: Entering directory `/dev/shm/Molden/7.1/GCCcore-11.3.0/molden7.1/src/ambfor'
gfortran -g -m64 -o ambfor ambfor.o qvdw.o alloc.o optimise.o lbfgs.o angle.o bond.o charge.o conn.o enegrd.o getinp.o param.o tors.o vdwlj.o math.o switch.o wrtout.o pbc.o bsolv.o -lm
/hpc/software/easybuild/software/binutils/2.38-GCCcore-11.3.0/bin/ld: switch.o: in function `allerd_':
/home/gijs/molden7.1/src/ambfor/switch.f:450: undefined reference to `_ZGVbN4v_expf'
/hpc/software/easybuild/software/binutils/2.38-GCCcore-11.3.0/bin/ld: /home/gijs/molden7.1/src/ambfor/switch.f:450: undefined reference to `_ZGVbN4v_expf'
/hpc/software/easybuild/software/binutils/2.38-GCCcore-11.3.0/bin/ld: /home/gijs/molden7.1/src/ambfor/switch.f:450: undefined reference to `_ZGVbN4v_expf'
/hpc/software/easybuild/software/binutils/2.38-GCCcore-11.3.0/bin/ld: /home/gijs/molden7.1/src/ambfor/switch.f:450: undefined reference to `_ZGVbN4v_expf'
/hpc/software/easybuild/software/binutils/2.38-GCCcore-11.3.0/bin/ld: /home/gijs/molden7.1/src/ambfor/switch.f:450: undefined reference to `_ZGVbN4v_expf'
/hpc/software/easybuild/software/binutils/2.38-GCCcore-11.3.0/bin/ld: switch.o:/home/gijs/molden7.1/src/ambfor/switch.f:450: more undefined references to `_ZGVbN4v_expf' follow
collect2: error: ld returned 1 exit status
make[1]: *** [ambfor] Error 1
make[1]: Leaving directory `/dev/shm/Molden/7.1/GCCcore-11.3.0/molden7.1/src/ambfor'
make: *** [ambfor/ambfor] Error 2
 (at easybuild/software/EasyBuild/4.7.2/lib/python3.6/site-packages/easybuild/tools/run.py:643 in parse_cmd_output)
== 2023-06-18 14:54:14,806 build_log.py:267 INFO ... (took 5 mins 5 secs)
== 2023-06-18 14:54:14,815 filetools.py:2015 INFO Removing lock /mnt/drosjfs/software/easybuild/software/.locks/_mnt_drosjfs_software_easybuild_software_Molden_7.1-GCCcore-11.3.0.lock...
== 2023-06-18 14:54:14,873 filetools.py:384 INFO Path /mnt/drosjfs/software/easybuild/software/.locks/_mnt_drosjfs_software_easybuild_software_Molden_7.1-GCCcore-11.3.0.lock successfully removed.
== 2023-06-18 14:54:14,873 filetools.py:2019 INFO Lock removed: /mnt/drosjfs/software/easybuild/software/.locks/_mnt_drosjfs_software_easybuild_software_Molden_7.1-GCCcore-11.3.0.lock
== 2023-06-18 14:54:14,874 easyblock.py:4257 WARNING build failed (first 300 chars): cmd " make  -j 1 CC="$CC" FC="$F90" FFLAGS="$FFLAGS -fallow-argument-mismatch" all" exited with exit code 2 and output:
gfortran -O2 -ftree-vectorize -march=native -fno-math-errno -fallow-argument-mismatch  -c -o src/atomdens.o src/atomdens.f
gfortran -O2 -ftree-vectorize -march=native -fno-math-err
== 2023-06-18 14:54:14,876 easyblock.py:328 INFO Closing log for application name Molden version 7.1
== 2023-06-18 14:54:14,877 build_log.py:267 INFO FAILED: Installation ended unsuccessfully (build directory: /dev/shm/Molden/7.1/GCCcore-11.3.0): build failed (first 300 chars): cmd " make  -j 1 CC="$CC" FC="$F90" FFLAGS="$FFLAGS -fallow-argument-mismatch" all" exited with exit code 2 and output:
gfortran -O2 -ftree-vectorize -march=native -fno-math-errno -fallow-argument-mismatch  -c -o src/atomdens.o src/atomdens.f
gfortran -O2 -ftree-vectorize -march=native -fno-math-err (took 5 mins 8 secs)
== 2023-06-18 14:54:14,886 build_log.py:267 INFO Results of the build can be found in the log file(s) /tmp/eb-av367krw/easybuild-Molden-7.1-20230618.144906.uNRzC.log
== 2023-06-18 14:54:14,900 build_log.py:171 ERROR EasyBuild crashed with an error (at easybuild/software/EasyBuild/4.7.2/lib/python3.6/site-packages/easybuild/base/exceptions.py:126 in __init__): build failed (first 300 chars): cmd " make  -j 1 CC="$CC" FC="$F90" FFLAGS="$FFLAGS -fallow-argument-mismatch" all" exited with exit code 2 and output:

```
do somebody know the reason.

**Comment by boegel** [comment]
@yanchenmochen If you still need help with this, please open an issue, we can't follow up on problems in merged PRs.

**Pull request merged**